client/dist/index.html

```diff
@@ -1,27 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <script src='https://meet.jit.si/external_api.js'></script>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script src="https://meet.jit.si/external_api.js"></script>
     <script src="main.js"></script>
+    <script src="page.js"></script>
     <script src="config.js"></script>
+    <style>
```

Comment:
We may eventually move all this style into a file called style.css (or whatever you wanna call it).  This will allow us to keep the styles separate from the content.